lamindb_setup/core/_hub_client.py

```diff
@@ -62,6 +62,8 @@ def connect_hub(
     fallback_env: bool = False, client_options: ClientOptions = ClientOptions()
 ) -> Client:
     env = Environment(fallback=fallback_env)
+    client_options.auto_refresh_token = False
+    client_options.persist_session = False
```

Comment:
Ah, I see!

Reply:
Without sign out supabase tried to reuse the session between environments and it looks like anon tokens for different environments have different format, so the session tried to reuse sessions between environments and it resulted in a n error.

Reply:
There is a problem that a session never expires so it sill hang forever in the auth table if we don't sign out and don't configure expiration time for **sessions** (not for access tokens).